neo/io/nixio.py

```diff
@@ -80,6 +82,38 @@ def calculate_timestamp(dt):
     return int(dt)
 
 
+def check_nix_version():
+    if not HAVE_NIX:
+        raise Exception(
+            "Failed to import NIX. "
+            "The NixIO requires the Python package for NIX "
+            "(nixio on PyPi). Try `pip install nixio`."
+
+        )
+
+    minverstr = ".".join(map(str, MIN_NIX_VER))
+    nixverstr = nix.__version__
+    nixver = nixverstr.split(".")
+
+    try:
```

Comment:
Would it makes sense to use `packaging.version` for the version handling here instead of doing explicit string handling?

Reply:
I considered it and since the `packaging` module isn't in the standard library, I didn't want to add an extra dependency. I only now realise that it's part of the setuptools and pip group of modules so it's pretty close to standard.

I'll amend.